distribution/docs/src/main/resources/_contents/_data/security-attributes-table-contents.adoc

```diff
@@ -0,0 +1,26 @@
+﻿
+.[[_security_attributes_table]]Security: Attributes in this group relate to security of the resource and metadata.
+[cols="1,2,1,1,1" options="header"]
+|===
+
+|Term
+|Definition
+|Datatype
+|Constraints
+|Example Value
+
+|security.access-groups
+|Attribute name for storing groups to enforce access
+controls upon
+|String
+|
+|
+
+|security.access-individuals
+|Attribute name for storing the email addresses of users to
```

Comment:
They are not comma delimited internally. It is actually a serialized list of values. I realize this is confusing and contradictory to how they are often treated in the Admin UI.